security-policy.md

```diff
@@ -4,4 +4,166 @@ title: Security Policy
+- **Affected Project:** Project URL or Hex.pm Package Name.
+- **Affected Version(s):** Specific versions (e.g., `v1.2.3`) or commit SHAs.
+- **Affected Files/Paths:** Specific source files or paths involved.
+- **Configuration Details:** Any special configurations required to reproduce
+  the issue.
+- **Reproduction Steps:** Step-by-step instructions to replicate the
+  vulnerability.
+- **Proof-of-Concept or Exploit Code:** If available, code that demonstrates the
+  vulnerability.
+- **Impact Assessment:** Potential impact and how the issue might be exploited
+  by an attacker.
+- **Your Contact Information:** Optional, but helpful if we need further
+  clarification.
+
+## 5. Dispute Handling
+
+### 5.1. Dispute Resolution Process
+
+- **Acknowledgment:** We will acknowledge disputes within **three business
+  days**.
+- **Initial Decision:** Within **one week** after acknowledgment, we will
+  provide a decision or request an extension.
+- **Extended Review:** If extended, we will make a final decision within **14
+  days**.
+- **Internal Review:** Disputes will trigger an internal referendum as per our
+  [Governance Documentation](https://github.com/erlef-cna/.github/blob/main/GOVERNANCE.md).
+
+### 5.2. Escalation
+
+- If the disputing party is unsatisfied with our final decision, they may
+  escalate the matter to our Root CNA, Red Hat, initiating a new dispute cycle.
+
+## 6. References to Root CNA Policies
+
+We adhere to the following policies and guidelines:
+
+- **Red Hat CNA Vulnerability Disclosure Policy:**
+  [Red Hat's Policy](https://access.redhat.com/articles/red_hat_cna_vulnerability_disclosure_policy)
+- **CVE Numbering Authority Operational Rules:**
+  [CNA Rules](https://www.cve.org/ResourcesSupport/AllResources/CNARules/#section_4-4_CNA_Judgment)
+- **CVE Program Policy and Procedure for Disputing a CVE Record:**
+  [Policy and Procedure](https://www.cve.org/Resources/General/Policies/CVE-Record-Dispute-Policy.pdf)
+
+## 7. Publication of CVEs
+
+### 7.1. CVE Publication
+
+- **CVE Database:** All assigned CVEs will be published through the official CVE
+  channels.
+- **Project Website:** CVEs will also be listed on our website at
+  [cna.erlef.org/cves](https://cna.erlef.org/cves).
+
+### 7.2. Notification
+
+- **Community and Stakeholders:** We will notify relevant parties through our
+  publication process. Project maintainers are responsible for additional communications to their users.
+
+## Additional Information
+
+### Handling of `SECURITY.md` Files
+
+- **Compliance:** We will adhere to the instructions in a project's
+  `SECURITY.md` file whenever feasible.
+- **Limitations:** We will only follow the project's security policies if the
+  maintainers respond within **14 days** of our initial communication. We will
+  not honor disclosure timelines exceeding our maximum timeframes and will
+  proceed with publishing zero-day vulnerabilities promptly.
```

Comment:
Should this be referred to as a public vulnerability as discussed above ?

Reply:
@IngelaAndin Yes. I will replace all references 👍